acceptance_tests/features/steps/edit_collection_exercise_details.py

```diff
@@ -0,0 +1,33 @@
+from behave import given, when, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import edit_collection_exercise_details_form, collection_exercise_details, \
+    collection_exercise
+
+
+@given('the internal user is on the collection exercise details page')
+def check_user_on_collection_exercise_details_page(_):
+    collection_exercise_details.go_to('RSI', '201808')
+    assert "023 RSI 201808 | Surveys | Survey Data Collection" in browser.title
+
+
+@when('they request to edit/amend collection exercise details')
+def navigate_to_edit_collection_exercise_details_page(_):
+    collection_exercise_details.click_edit_collection_exercise_period_button()
+
+
+@when('they edit/amend the details')
+def edit_collection_exercise_details(_):
+    edit_collection_exercise_details_form.edit_period('201906')
+    edit_collection_exercise_details_form.edit_user_description('12 June 2019')
+    edit_collection_exercise_details_form.click_save()
+
+
+@then('they can view the updated details')
+def view_updated_collection_exercise_details(context):
+    collection_exercises = collection_exercise.get_collection_exercises()
```

Comment:
Does there need to be a wait here or are the changes from CE instantaneous?

Regardless, the tests pass locally when running against the branched services 👍

Reply:
It seemed to be instantaneous when I was testing it so I don't believe it needs a wait